src/main/java/com/example/bookstore/controller/BookController.java

```diff
@@ -25,30 +31,38 @@ public class BookController {
 
     @GetMapping
     @ResponseStatus(HttpStatus.OK)
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "get all books", description = "get list of available books")
+    public List<BookDto> getAll(Pageable pageable,
+                                @RequestParam(defaultValue = "1") int page,
+                                @RequestParam(defaultValue = "5") int size) {
+        Pageable updatedPageable = PageRequest.of(page - 1, size, pageable.getSort());
+        return bookService.findAll(updatedPageable);
     }
 
     @GetMapping("/{id}")
     @ResponseStatus(HttpStatus.OK)
+    @Operation(summary = "get book by id", description = "get book by id if available")
     public BookDto getById(@PathVariable Long id) {
         return bookService.getById(id);
     }
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
+    @Operation(summary = "create a new book", description = "create a new book")
```

Comment:
We can use only a summary or description if the text is the same. Please fix it for every method